runtime/runtime/src/ext.rs

```diff
@@ -70,7 +70,7 @@ impl<'a> RuntimeExt<'a> {
         epoch_id: EpochId,
         prev_block_hash: CryptoHash,
         last_block_hash: CryptoHash,
-        epoch_info_provider: &'a (dyn EpochInfoProvider),
+        epoch_info_provider: &'a dyn EpochInfoProvider,
```

Comment:
just out of curiosity, why this? Is it a real change or just formatting?

Reply:
Just formatting, the brackets looked a weird 😅